Example/Core/Tests/FIRAppTest.m

```diff
@@ -258,13 +258,13 @@ - (void)testDeleteApp {
 }
 
 - (void)testErrorForSubspecConfigurationFailure {
-  NSError *error = [FIRApp errorForSubspecConfigurationFailureWithDomain:kFirebaseAdMobErrorDomain
-                                                               errorCode:FIRErrorCodeAdMobFailed
-                                                                 service:kFIRServiceAdMob
+  NSError *error = [FIRApp errorForSubspecConfigurationFailureWithDomain:kFirebaseCoreErrorDomain
+                                                               errorCode:-38
+                                                                 service:kFIRServiceAuth
```

Comment:
I guess we don't actually validate the service here?

Reply:
Right.